src/web/app/components/sortable-table/sortable-table.component.html

```diff
@@ -1,26 +1,29 @@
-<table class="table table-bordered table-striped margin-bottom-20px">
```

Comment:
Seems that removing this line has caused some complications with the other components using the `sortable-table`:
<img width="1200" alt="Screenshot 2023-07-23 at 12 48 12 AM" src="https://github.com/TEAMMATES/teammates/assets/48304907/54cbd748-1e14-45e5-ba31-5e37ac841f5e">

Think it affects most of the question statistics components... so either
- Add the `margin-bottom-20px` back in, and handle it separately in the instructor home and sessions pages
- Make it another customisable part of the `sortable-table` component...
- Add the margins back in each of the other question statistics components (and remove the now redundant styling from the `sortable-table.component.scss` file)

Not too sure which is the best approach, though I think the last one is easiest and it also makes sense that the margin after the table is left to each component to handle.

Reply:
I agree with the last approach as well! I will make the changes shortly 👍🏻